packagingbuild/bionic/Dockerfile

```diff
@@ -1,26 +1,26 @@
 FROM buildpack-deps:bionic
+ARG assets_path
+ENV ASSETS_PATH $assets_path
 
 # Make noninteractive setting permanent
 ENV DEBIAN_FRONTEND noninteractive
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections -v
 
 # Enable remote pubkey access
-RUN mkdir /root/.ssh && chmod 700 /root/.ssh && \
-    echo "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCdCmmPjsOBWRXc+PKdgDRrsciNjp25zTacyz8Gdkln2ma046brOYXAphhp/85DKgHtANBBt3cl4+HnpDbmAfyq2qZT7hWzAbMxtq0Sj+yyFyUdreXoe4gEKyxpV6o8p/R/XzEcawvqX/vFc5EIFmvTdamxZs9DQmOE5AZMzUB18Kerkrb0/arUcZ8iMi9Ng9a18avow+7oUFZ6Oub7ISz/dkIRojaKO/2paJZ4p+v7ZLn7Hq8TUeBkgAlx872oh8J8linhIq17zK6x4MGL8qiurp2hnfe0cuCxwcsYGy+4DfK51+E2vks6FprCIfF5hIdz26euPn67/YpM0F0b5nXF busybee@drone" >> /root/.ssh/authorized_keys
+RUN mkdir /root/.ssh && chmod 700 /root/.ssh
+COPY $ASSETS_PATH/busybee.pub /root/.ssh/authorized_keys
```

Comment:
Can you please revert the key to a hardcoded one as before for all the Dockerfiles instead of relying on additional ENV variables?

That'll fix the builds.